backend/corpora/common/corpora_orm.py

```diff
@@ -247,13 +247,12 @@ class DbDataset(Base, AuditMixin):
     tombstone = Column(Boolean, default=False, nullable=False)
     original_id = Column(String)
     published = Column(Boolean, default=False)
+    explorer_s3_uri = Column(String)
```

Comment:
I'm curious -- in the upload processing step, we create a row for each artifact created (i.e. H5AD, RDS, and LOOM), but not CXG. And yet we have a CXG `DatasetArtifactFileType` type. Why was that decision made and why link the artifact s3_uri here instead of populating another row in the DatasetArtifact table with the CXG type? Linking it here seems to violate the semantics of that table's purpose.

Reply:
Discussed above https://github.com/chanzuckerberg/corpora-data-portal/pull/1216#discussion_r645315885